fvt/predictor_test.go

```diff
@@ -119,6 +119,7 @@ var _ = Describe("Predictor", func() {
 					"value": "0",
 				},
 			},
+			"podsPerRuntime": 1,
```

Comment:
I'm assuming these added to avoid hitting the GH Actions environment resource constraints? Are the SR pods in the controller namespace still up  when  the user-namespace tests are run?

Reply:
Yes. The SR pods are still up because the predictor test config has scaleToZero=false in order to prevent pods flapping as
			// Predictors are created and deleted